### DISCUSSION TITLE
DataStorage 질문입니다.

### DISCUSSION THREAD

**Discussion by Sonak0930** [naverz/zepeto-studio-kor#1088] [5. Q&A]
![image](https://user-images.githubusercontent.com/59792911/181174756-19189bc0-7ac6-4e56-bbc8-6ed332865621.png)
다음과 같은 방식으로 숫자를 저장하고 불러오는데는 성공했습니다.
그런데 숫자말고, 객체나 오브젝트는 저장할 수 없을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, Datastorage는 데이터를 저장하기 위한 공간이어서 
저장된 데이터를 보고 오브젝트를 생성하는 방식을 추천 드립니다.
